### PR TITLE
Improve Docs.

### PR DESCRIPTION
**Pull request by Dushyantsingh-ds**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,11 +2,11 @@
 
 This repository contains built-in samples of Azure Policies that can be used as reference for creating and assigning policies to your subscriptions and resource groups. For easy search of all built-in  with descriptions, see [Policy samples](https://docs.microsoft.com/azure/governance/policy/samples/) on docs.microsoft.com.
 
-For custom policy samples, check out our Community repo! (https://github.com/Azure/Community-Policy)
+For custom policy samples, [check](https://github.com/Azure/Community-Policy) out our Community repo!
```
Comment (pilor) [inline README.md:5]: Thanks for this but please put the link on the same words as below (our Community repo!).

**Review by pilor — CHANGES_REQUESTED** [review]

**Review by Dushyantsingh-ds — COMMENTED** [review]
I have done the changes.

File: README.md [hunk 2]
```diff
@@ -2,11 +2,11 @@
 
 This repository contains built-in samples of Azure Policies that can be used as reference for creating and assigning policies to your subscriptions and resource groups. For easy search of all built-in  with descriptions, see [Policy samples](https://docs.microsoft.com/azure/governance/policy/samples/) on docs.microsoft.com.
 
-For custom policy samples, check out our Community repo! (https://github.com/Azure/Community-Policy)
+For custom policy samples,out [our Community repo!](https://github.com/Azure/Community-Policy)
```
Comment (pilor) [inline README.md:5]:
```suggestion
For custom policy samples, check out [our Community repo!](https://github.com/Azure/Community-Policy)
```

**Review by pilor — CHANGES_REQUESTED** [review]

**Comment by Jazzette13** [comment]
Trying to transfer three iPhone

**Review by Jazzette13 — APPROVED** [review]

**Comment by kenieva** [comment]
Removing due to outdated. Will make changes in next publication. Thank you.

**Pull request closed without merging**